src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -31,7 +31,7 @@ public class RazorViewEngineOptions
         /// locate Razor files.
         /// </summary>
         /// <remarks>
-        /// At startup, this is initialized to include an instance of <see cref="PhysicalFileProvider"/> that is
+        /// At startup, this is initialized to include an instance of PhysicalFileProvider that is
```

Comment:
We use `IHostingEnvironment.ContentRootFileProvider`, so I guess Hosting sorts it out for us.